schemas/mod-source-record-manager/jobMonitoring.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Job Monitoring Schema",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "UUID",
+      "$ref": "../common/uuid.json"
+    },
+    "jobExecutionId": {
+      "description": "Corresponding jobExecution id",
+      "type": "string"
```

Comment:
Should be UUID also

Reply:
Good point, they use String sometimes but Uuid is more convenient